config.lua.dist

```diff
@@ -20,6 +20,7 @@ expFromPlayersLevelRange = 75
 -- NOTE: maxPlayers set to 0 means no limit
 -- NOTE: allowWalkthrough is only applicable to players
 -- NOTE: two-factor auth requires token and timestamp in session key
+-- NOTE: statusCountMaxPlayersPerIp allows you to only count first X players per IP in status response (0 = disabled)
```

Comment:
Might be me being overly pedantic but the wording here seems of. Doesn't really matter if it's the first X or last X or any X:


```suggestion
-- NOTE: statusCountMaxPlayersPerIp allows you to only count up to X players per IP in status response (0 = disabled)
```

Feel free to mark as resolved if you'd rather ignore 😆